tomcat/build.go

```diff
@@ -66,6 +66,10 @@ func (b Build) Build(context libcnb.BuildContext) (libcnb.BuildResult, error) {
 
 	b.Logger.Title(context.Buildpack)
 
+	if b.SBOMScanner == nil {
```

Comment:
Do we need to change this location still? Not sure it matters, but if it's not strictly necessary it would slim down the PR a bit.